.github/workflows/rc_sync.yml

```diff
@@ -51,17 +52,19 @@ jobs:
           git checkout -b "$BRANCH"
           # Exclude some files known to be divergent during the release process from the PR
           git checkout master -- pennylane/_version.py
-          git checkout master -- doc/releases/changelog-dev.md
-          if ! git diff-index --quiet HEAD; then
-            git config user.name "GitHub Actions Bot"
-            git config user.email "<>"
-            git commit -m "exclude files from pr"
+          git checkout master -- doc/development/release_notes.md
```

Comment:
This will now assume that we've renamed `doc/releases/changelog-dev.md` to `doc/releases/changelog-XX.md` before the `v0.XX.rc0` branch has been created, correct?